package.json

```diff
@@ -146,11 +139,6 @@
         "title": "CleanAll",
         "category": "XMake"
       },
-      {
-        "command": "xmake.onBuildRun",
-        "title": "BuildRun",
-        "category": "XMake"
-      },
```

Comment:
why?

Reply:
`onBuild` is called before running the target if `onRunMode` is set to `buildRun`